### PR TITLE
clarifying Getting Started

### PR DESCRIPTION
**Pull request by beckettsean**

### CODE REVIEW

**Comment by gunnaraasen** [comment]
+1

**Comment by toddboom** [comment]
+1

File: content/docs/v0.9/introduction/getting_started.md [hunk 1]
```diff
@@ -131,4 +133,4 @@ InfluxDB supports a sophisticated query language, allowing many different types
 > SELECT * FROM cpu_load_short WHERE value > 0.9
 ```
 
-This is all you need to know to write data into InfluxDB and query it back. Of course, to write significant amounts of data you will want to access the HTTP API directly, or use one of the many client libraries. 
+This is all you need to know to write data into InfluxDB and query it back. Of course, to write significant amounts of data you will want to access the HTTP API directly, or use one of the many client libraries.
```
Comment (corylanou) [inline content/docs/v0.9/introduction/getting_started.md]: do we have a link to the HTTP API yet?

Comment (beckettsean) [inline content/docs/v0.9/introduction/getting_started.md]: Not yet.

**Pull request merged**